tests/sdk/nodejs/metadata/step2/index.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2016-2022, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as pulumi from "@pulumi/pulumi";
```

Comment:
It's helpful to add a comment on subsequent steps explaining what changed about the program. I think these steps are identical?

Reply:
I agree and will add some.
Step 2 has different values for the readonly fields, and the test expects no changes in step 2.